src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -51,7 +53,7 @@ public static class BitOperations
         /// <param name="value">The value.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         [CLSCompliant(false)]
-        public static bool IsPow2(uint value) => (value & (value - 1)) == 0 && value != 0 ;
+        public static bool IsPow2(uint value) => (value & (value - 1)) == 0 && value != 0;
```

Comment:
I'm not sure right now why this change was made (probably automatically by VS), but it doesn't hurt I think.